src/wallet/wallet_tx_builder.cpp

```diff
@@ -245,48 +248,44 @@ CalcZIP317Fee(
         });
     }
 
-    size_t logicalActionCount;
+    std::vector<CTxIn> vin{};
+    size_t sproutInputCount = 0;
+    size_t saplingInputCount = 0;
+    size_t orchardInputCount = 0;
     if (inputs.has_value()) {
-        std::vector<CTxIn> vins{};
-        for (const auto& utxo : inputs.value().utxos) {
-            vins.emplace_back(
+        for (const auto& utxo : inputs->utxos) {
```

Comment:
I would normally eschew `->` on a `std::optional`, but here I agree it's fine.

Reply:
Ah, whoops. I had assumed it was identical to `value()`. I think I’m inclined to change it back then.